doc/classes/@GlobalScope.xml

```diff
@@ -2316,41 +2316,41 @@
 		<constant name="KEY_Z" value="90" enum="Key">
 			Z key.
 		</constant>
-		<constant name="KEY_BRACKETLEFT" value="91" enum="Key">
-			[ key.
+		<constant name="KEY_BRACKETLEFT" value="91" enum="Key" keywords="KEY_SQUARE_BRACKET_LEFT">
+			Left bracket ([code][[/code]) key.
 		</constant>
 		<constant name="KEY_BACKSLASH" value="92" enum="Key">
-			\ key.
+			Backslash ([code]\[/code]) key.
 		</constant>
-		<constant name="KEY_BRACKETRIGHT" value="93" enum="Key">
-			] key.
+		<constant name="KEY_BRACKETRIGHT" value="93" enum="Key" keywords="KEY_SQUARE_BRACKET_RIGHT">
+			Right bracket ([code]][/code]) key.
 		</constant>
-		<constant name="KEY_ASCIICIRCUM" value="94" enum="Key">
-			^ key.
+		<constant name="KEY_ASCIICIRCUM" value="94" enum="Key" keywords="KEY_CARET">
+			Caret ([code]^[/code]) key.
 		</constant>
-		<constant name="KEY_UNDERSCORE" value="95" enum="Key">
-			_ key.
+		<constant name="KEY_UNDERSCORE" value="95" enum="Key" keywords="KEY_UNDERLINE">
+			Underscore ([code]_[/code]) key.
 		</constant>
-		<constant name="KEY_QUOTELEFT" value="96" enum="Key">
-			` key.
+		<constant name="KEY_QUOTELEFT" value="96" enum="Key" keywords="KEY_BACKTICK, KEY_BACKQUOTE">
+			Backtick ([code]`[/code]) key.
 		</constant>
-		<constant name="KEY_BRACELEFT" value="123" enum="Key">
-			{ key.
+		<constant name="KEY_BRACELEFT" value="123" enum="Key" keywords="KEY_CURLY_BRACKET_LEFT">
+			Left brace ([code]{[/code]) key.
 		</constant>
-		<constant name="KEY_BAR" value="124" enum="Key">
-			| key.
+		<constant name="KEY_BAR" value="124" enum="Key" keywords="KEY_PIPE">
+			Vertical bar ([code]|[/code]) key.
```

Comment:
```suggestion
			Vertical bar or pipe ([code]|[/code]) key.
```
Potentially?

For some reason it seems weird to not use the name as it appears in the enum itself. But I also see the logic of only including one name for the character, and choosing the most popular one, even if it's not the one in the enum itself.

Reply:
I was going to add things along the lines of "also known as...", but I believed it to be out of scope of this PR.